docs/using.md

```diff
@@ -384,14 +385,26 @@ If your workload requires a MongoDB connection (most do), then you can pass it i
 
 ## Outputs
 
-Genny's primary output is time-series data. Every time an Actor performs an operation, such as an insert, a removal, a runcommand, etc, the Actor thread starts a timer. When the operation returns from the server, the operation is recorded as either a success or failure. The duration of the operation, as viewed from the client, is recorded with the operation completion time.
+Genny's primary output is time-series data. Every time an Actor performs an operation, such as an
```

Comment:
I reflowed the content to 100 columns max, which seems like a good practice to avoid super long lines. I opted not to do this for the whole file, though.

Reply:
Ok. I'm definitely not a fan of super long lines, but makes sense that reflowing the entire set of markdown files in this repo should be done as a separate change. I'll revert the reflowing.